src/Database/DBA.php

```diff
@@ -251,22 +255,20 @@ private static function logIndex($query) {
 	}
 
 	public static function escape($str) {
-		if (self::$connected) {
-			switch (self::$driver) {
-				case 'pdo':
-					return substr(@self::$connection->quote($str, PDO::PARAM_STR), 1, -1);
-				case 'mysqli':
-					return @self::$connection->real_escape_string($str);
-			}
-		} else {
```

Comment:
Have you checked whether the removal of this part could make a problem, possibly during the installation phase?

Reply:
Huh? Maybe a mistake or something strange. 👍

Reply:
And good catch!